mne/io/meas_info.py

```diff
@@ -1211,3 +1214,141 @@ def _empty_info():
     info['nchan'] = info['sfreq'] = 0
     assert set(info.keys()) == set(RAW_INFO_FIELDS)
     return info
+
+
+class DigMontage(object):
+    """Montage for Digitized data
+
+    Montages are typically loaded from a file using read_montage. Only use this
+    class directly if you're constructing a new montage.
+
+    Parameters
+    ----------
+    hsp : array, shape (n_points, 3)
+        The positions of the channels in 3d.
```

Comment:
Polhemus, which is one of the most common digitizer outputs this as a txt file. The read_dig_montage handles whether you specified it as a filename or array and passes it to the class as an array.

the naming convention comes from the MNE manual:
```
--hsp <filename>
The name of the file containing the locations of the fiducials and additional points on the head surface. This file is optional.
--hpi <filename>
The name of a text file containing the locations of the HPI coils in the MEG device coordinate frame, given in millimeters. The order of the coils in this file does not have to be the same as that in the elp file. This option is mandatory.
--elp <filename>
The name of the file containing the locations of the fiducials and the HPI coils. This option is mandatory.
```